### DISCUSSION TITLE
DockerHub images are 7 months out of date

### DISCUSSION THREAD

**Discussion by nrandon** [abraunegg/onedrive#1886] [Q&A] [answered]
Can we have new DockerHub images pushed ?

**Comment by abraunegg** [comment]
@nrandon 

> Can we have new DockerHub images pushed ?

Unfortunately not .. Docker changed their policy for OpenSource integrations from GitHub. I have been trying to get a different build | integration working, however nothing is working correctly.

So for the moment - the *only* solution is for you to build your own Docker container. Please complain to Docker.

**Comment by abraunegg** [comment, marked as answer]
@nrandon 
With the release of v2.4.18 - Docker containers are being automatically built and pushed to Docker Hub. Please check [docker.md](https://github.com/abraunegg/onedrive/blob/master/docs/Docker.md) for updated Docker Tag details.
